public/video-ui/src/components/FormFields/DatePicker.js

```diff
@@ -57,19 +61,23 @@ function MinuteSelector({ date, onChange }) {
 }
 
 function DateSelector({ date, onChange }) {
+  const minDate = moment().toDate();
   const datePickerParams = {
     className: 'form__field',
-    selected: date,
-    minDate: moment(),
+    selected: date ? date.toDate() : null,
+    minDate,
     dateFormat: DATE_FORMAT,
-    readOnly: true,
+    readOnly: false,
     onChange: newDate => {
       const base = date ? date : moment().hours(0).minutes(0);
-      onChange(newDate.hours(base.hours()).minutes(base.minutes()));
+      const onChangeDate = moment(newDate)
+        .hours(base.hours())
+        .minutes(base.minutes());
+      onChange(onChangeDate);
     }
   };
 
-  return <Picker {...datePickerParams} />;
+  return <DatePicker {...datePickerParams} />;
```

Comment:
`react-datepicker` uses Date objects to work, or at least the examples suggest this: https://reactdatepicker.com/

Using `moment()` types didn't work after the upgrade, so these changes convert moment to date types when passing to `DatePicker`